submissions/Levi123/memory-pair-game/script.js

```diff
@@ -0,0 +1,142 @@
+const SELECTORS = {
+    flipContainer : document.querySelector('.flip-container'),
+    content : document.querySelector('.main-content__playing-field'),
+    textContent: document.querySelector('.main-content'),
+    timer : document.querySelector('.seconds'),
+    flipCounter: document.querySelector('.flips'),
+}
+
+const STATE = {
+    totalFlips : 0,
+    totalTime : 0,
+    flipElementIdArray : [],
+    cardOne : undefined,
+    cardTwo : undefined,
```

Comment:
```suggestion
    cardOne : null,
    cardTwo : null,
```

Reply:
@madmaxWMFU thank you, I understood null it's - the value of the absence of an object, while undefined - uncertainty.